foxglove_bridge_base/include/foxglove_bridge/server_interface.hpp

```diff
@@ -3,6 +3,7 @@
 #include <functional>
```

Comment:
README.md needs to be updated. Info about how and why different frame_ids can pass through the throttle